mdal/mdal_driver_manager.cpp

```diff
@@ -240,5 +241,22 @@ MDAL::DriverManager::DriverManager()
 #if defined HAVE_HDF5 && defined HAVE_XML
   mDrivers.push_back( std::make_shared<MDAL::DriverXdmf>() );
 #endif
+
+  // TODO : defined in cmake a variable to define the need to load dynamic driver
+  loadDynamicDrivers();
+}
+
+
+void MDAL::DriverManager::loadDynamicDrivers()
+{
+  // TODO: some logic to look on particular folder if there is any library files and try to create it
```

Comment:
also consider
1. environment variable
2. hardcoded path relative to install path (e.g /usr/local/share/mdal/drivers)